src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/VsTelemetryService.cs

```diff
@@ -39,5 +161,88 @@ public TelemetryEventCorrelation Report(string eventName, string description, Ex
                 exceptionObject: exception,
                 gatherEventDetails: callback);
         }
+
+        /// <summary>
+        /// Hashes personally identifiable information for telemetry consumption.
+        /// </summary>
+        /// <param name="value">Value to hashed.</param>
+        /// <returns>Hashed value.</returns>
+        public string HashValue(string value)
+        {
+            using (var crypto = SHA256.Create())
```

Comment:
Does this return the native or managed version of SHA256? The managed version is not FIPS compliant.